### PR TITLE
Rewrite: modular approach

### LINKED ISSUE #3: Separate index.go in several files
Seems like the way to go is to move the Telegram and Github content to folders that will act as modules, however: I tried that while making the [0.0.1 PR](https://github.com/berserktech/telebot/pull/1) and it didn't work. Probably because of my $GOPATH or something environment-related.

### PR DESCRIPTION
**Pull request by ghostbar**
Fixes #3

### CODE REVIEW

File: gh/content.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	return fmt.Sprintf(
+		"%s %s the %s: %s %s%s",
+		s.Link(), c.Action, kind, c.Title, c.HTMLURL, body,
+	)
+}
+
+// NotAllowed returns an error if the received Action is not valid.
+func (c Content) NotAllowed() error {
+	switch c.Action {
+	case "labeled",
```
Comment (sadasant) [inline gh/content.go:28]: Awesome!

File: gh/comment.go [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package gh
```
Comment (sadasant) [inline gh/comment.go:1]: Can you make this more explicit? as in `github`? Same with `telegram`. Please 🙏❤️

**Review by sadasant — CHANGES_REQUESTED** [review]

File: gh/comment.go (hunk 2 above)
Comment (ghostbar) [reply gh/comment.go:1]: The github library already calls itself github. That would create some issues. Better avoid it and leave it as `gh` and `tg` :-)

File: gh/content.go (hunk 1 above)
Comment (ghostbar) [reply gh/content.go:28]: :-D

**Comment by sadasant** [comment]
This is the problem that CircleCI is having: https://github.com/golang/go/issues/27056
I'm also having this myself. On it...

**Comment by sadasant** [comment]
IMPORTANT: This doesn't work on Zeit.co. They use an older version:
https://github.com/zeit/now-builders/blob/master/packages/now-go/go-helpers.js#L93
and they disable GO111MODULE by default due to having `GO111MODULE=auto`.

I can manually set `GO111MODULE` to `on`, but 1.11 doesn't know how to mix go.mod with dependencies in GOPATH, see: https://golang.org/doc/go1.12#modules

> ... the go command now supports module-aware operations outside of a module directory, provided that those operations do not need to resolve import paths relative to the current directory or explicitly edit the go.mod file. ...

I did the silliest of pull requests to [zeit/now-builders](https://github.com/zeit/now-builders) just to bring awareness to the issue: https://github.com/zeit/now-builders/pull/247

**Comment by ghostbar** [comment]
Going back to single-module repo for Zeit :-)

**Review by sadasant — APPROVED** [review]
🎉

**Review by betacar — APPROVED** [review]
💯

**Pull request merged**